lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/security/FileBasedSystemAccessControl.java

```diff
@@ -216,13 +214,16 @@ public SystemAccessControl create(Map<String, String> config)
             requireNonNull(config, "config is null");
 
             Bootstrap bootstrap = new Bootstrap(
-                    binder -> configBinder(binder).bindConfig(FileBasedAccessControlConfig.class));
+                    binder -> configBinder(binder).bindConfig(FileBasedAccessControlConfig.class),
+                    new FileBasedSystemAccessControlModule());
+
             Injector injector = bootstrap
                     .doNotInitializeLogging()
                     .setRequiredConfigurationProperties(config)
                     .initialize();
+
             FileBasedAccessControlConfig fileBasedAccessControlConfig = injector.getInstance(FileBasedAccessControlConfig.class);
-            String configFileName = fileBasedAccessControlConfig.getConfigFile().getPath();
+            String configFileName = fileBasedAccessControlConfig.getConfigFile();
```

Comment:
Can we add an validation to ensure that the `file` exists.

Reply:
done

Reply:
not really because we don't know if this is a file or http endpoint here. I renamed the variable to configFile so that is it less misleading